test/e2e/mcc_test.go

```diff
@@ -0,0 +1,111 @@
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/openshift/machine-config-operator/test/framework"
+	"github.com/openshift/machine-config-operator/test/helpers"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Test case for https://github.com/openshift/machine-config-operator/issues/358
+func TestMCCPausedKubeletCAAlert(t *testing.T) {
+
+	var testPool = "master"
```

Comment:
Does this have to target the `master` MachineConfigPool? I ask because a common pattern in the e2e test suite is that we create an ephemeral `infra` pool for the purpose of the test, assign a single worker node to that pool, then apply the MachineConfig. The reason is because we can apply the MachineConfig to a single node (and reset when we're done) much faster than rolling it out to the entire MachineConfigPool.

Here's an example: https://github.com/openshift/machine-config-operator/pull/2795/files#diff-32df39b541567d57b84246d0d48a44792350b5af97d060cd0e44c33949c07370R55-R96

Reply:
So I did originally have it set up the "infra"  way, but:

1. I flipped it to master when I was testing against single node (because if a node is member of multiple pools, master seems to supersede it). 
2. There is no escaping the certificate rotation -- all unpaused pools will get it -- I don't have a way to confine a certificate rotation to certain pools (well, aside from pause, which is what I am testing) 

It probably does save a little bit of time having just one node be paused, otherwise we'd have to wait once we unpause. 

I'll put it back to infra  (and maybe copy this test as it sits into the single node tests if we care?)

Reply:
I think for this particular test we don't really need to create infra pool because cert rotation would take place for all pools in the cluster. And for test purpose it is fine to pause only master pool if it helps with SNO case too.